bats_test/downloads/folder-downloads/cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIR="$(cd "$(dirname "${BATS_TEST_FILENAME}")" >/dev/null 2>&1 && pwd)"
+
+rm -rf \
+    "${DIR}/Files With a Variety of Extensions Public" \
```

Comment:
Why not just `${DIR}/*`?

Reply:
At `${DIR}/` there is the `.bats` test files